pkg/apis/core/types.go

```diff
@@ -434,7 +434,7 @@ type PersistentVolumeClaimSpec struct {
 	// that are lower than previous value but must still be higher than capacity recorded in the
 	// status field of the claim.
 	// +optional
-	Resources ResourceRequirements
+	Resources StorageRequirements
```

Comment:
should we be more explicit, and keep `resource` in the type name, like `VolumeResourceRequirements`?

Reply:
I don't have a strong opinion either way.

@msau42?